test/index.js

```diff
@@ -1,34 +1,38 @@
 "use strict"
-var test = require("tap").test
-var requireInject = require("require-inject")
+const tap = require('tap');
+const requireInject = require('require-inject');
 
-test("Windows", function (t) {
-  t.plan(1)
-  var hasUnicode = requireInject("../index.js", {
+tap.test("Windows", function (t) {
+  const hasUnicode = requireInject("../index.js", {
     os: { type: function () { return "Windows_NT" } }
-  })
-  t.is(hasUnicode(), false, "Windows is assumed NOT to be unicode aware")
-})
-test("Unix Env", function (t) {
-  var hasUnicode = requireInject("../index.js", {
+  });
+  t.equal(hasUnicode(), false, "Windows is assumed NOT to be unicode aware");
```

Comment:
I'm not a fan of the `equal` verb, tbh, I like my old-school, pre-Node tap verbs, tyvm. =p